graph/appender/appender.go

```diff
@@ -0,0 +1,13 @@
+package appender
+
+import "github.com/kiali/kiali/graph/tree"
+
+type Appender interface {
+	AppendGraph(trees *[]tree.ServiceNode, namespaceName string) error
+}
+
+func checkError(err error) {
+	if err != nil {
+		panic(err.Error)
```

Comment:
Not sure we really want to `panic`, say, in case of temporary network failure?

Reply:
The panics are handled in graph.handlePanic() and converted into an error response.  Is that a bad approach?  As an aside, this is not new for the PR so any requested changes should likely be handled separately.

Reply:
Ah, ok. Bad approach or not, I don't know, but if it's the way chosen before then it's ok. So it's more or less like an unchecked exception.

But since then I have one more little comment: the `error` in return call signature of `AppendGraph` are useless then and can be simplified

Reply:
+1, good catch, I will remove the error return.